JobHandlerApp/JobPlanner/IJob.cs

```diff
@@ -6,6 +6,8 @@ public interface IJob
     {
         bool IsFailed { get; set; }
 
+        DateTime StartJob { get; set; }
```

Comment:
дейтовые филды обычно постфиком таким обозначают
```suggestion
        DateTime StartJobAt { get; set; }
```